### PR TITLE
revamp call notification screen 

### PR DESCRIPTION
**Pull request by AndyScherzinger**
with 
* dark theme and 
* constraint layout

call

Beware that the system bar will still be blue. In order to get that part in black one would need to set another theme like `NoActionBar`

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -17,5 +17,6 @@
     <color name="nc_light_grey">#E8E8E8</color>
     <color name="grey_600">#757575</color>
     <color name="nc_grey">#D5D5D5</color>
+    <color name="white30">#e9ffffff</color>
```
Comment (mario) [inline app/src/main/res/values/colors.xml]: Upper case.

**Comment by AndyScherzinger** [comment]
lightblack

**Comment by AndyScherzinger** [comment]
lighter black/grey tone

**Comment by mario** [comment]
Thanks mate, as always!

**Comment by mario** [comment]
Will wait for build to turn green and merge.

**Pull request merged**